### DISCUSSION TITLE
[@types/node] node:sqlite missing options

### DISCUSSION THREAD

**Discussion by vassbence** [DefinitelyTyped/DefinitelyTyped#71369] [Issues with a @types package]
The types in `"@types/node": "^22.10.1"` for `DatabaseSync` are out of sync with the 22.X LTS documentation. https://nodejs.org/docs/latest-v22.x/api/sqlite.html#class-databasesync

[a comment by an automated account is not shown]

**Comment by Renegade334** [comment]
Merged in #71483.
